src/components/classroom/Classroom.js

```diff
@@ -137,7 +137,7 @@ class ClassroomModal extends Component {
 
     deleteClassroom = () => {
         const userClassrooms = this.props.classrooms.classrooms;
-        let optionItems = [];
+        const optionItems = [];
```

Comment:
Some of these might not actually be const. This could cause issues. I know eslint gave a warning but it's something to be careful when using --fix.